progress/.gitignore

```diff
@@ -0,0 +1 @@
+lib
```

Comment:
Should we put all of the `lib` ignores in the root of the repo or move `node_modules` into each sample's gitignore?

Reply:
I think we should have node_modules in each sample's `gitignore`. The idea is that each sample should be usable as a standalone, so we shouldn't rely on anything in the top-level dir.